resources/user-agents/browsers/firefox/firefox-49-52.json

```diff
@@ -30,7 +30,7 @@
           "match": "Mozilla/5.0 (#PLATFORM#) Gecko* Firefox/#MAJORVER#.#MINORVER#*",
           "device": "Linux Desktop",
           "platforms": [
-            "Ubuntu_64", "Ubuntu_12_04", "Ubuntu_32", "Ubuntu",
```

Comment:
This removed Ubuntu platform won't actually catch any useragents because the platform's pattern is defined as `*Ubuntu/12.04*` which doesn't seem to ever happen in this part of the UA (it does happen after the `Gecko` keyword in older versions of firefox.